src/pyOpenMS/pxds/IndexedMzMLDecoder.pxd

```diff
@@ -8,8 +8,41 @@ cdef extern from "<OpenMS/FORMAT/HANDLERS/IndexedMzMLDecoder.h>" namespace "Open
 
     cdef cppclass IndexedMzMLDecoder:
         IndexedMzMLDecoder() nogil except +
+            # wrap-doc:
```

Comment:
I think this is the whole class description, should probably go under the cdef cppclass - see MSExperiment.pxd

Reply:
Done